draft-meynell-panrg-scion-research-questions.md

```diff
@@ -214,9 +239,14 @@ There are multiple possible and independent solution steps here:
 * Allow queries from start AS to end AS across multiple segments. This should be very easy to implement and would be compatible with the current wire protocol (protobuf).
```

Comment:
I don't understand the approach. Can you clarify?

Reply:
I did. Can you have another look?